src/foam/nanos/auth/Region.js

```diff
@@ -43,13 +50,13 @@ foam.CLASS({
       name: 'getRegionCode',
       type: 'String',
       code: function() {
-        if ( this.code && this.code.length > 3 ) {
-          return this.code.substring(3, this.code.length);
+        if ( this.isoCode && this.isoCode.length > 3 ) {
```

Comment:
@nanopay-arthur  .. this does not work if isoCode is not set. Are you planning on setting the ISO code for all regions that are currently configured? Intuit code relies on the retrieveRegionCode function that properly maps code to 2 digit region. so I need this method to work.

Reply:
nevermind .. i see the proxy to getCode above now.

Reply:
@nanoscott if an explicit 'isoCode' value is not passed in, the 'code' value gets copied over to 'isoCode' via the factory.